package.json

```diff
@@ -24,6 +26,8 @@
     "gulp-mocha": "~2.2.0",
     "gulp-mocha-phantomjs": "~0.11.0",
     "gulp-rename": "~1.2.2",
+    "gulp-sequence": "^0.4.5",
+    "gulp-template": "^3.1.0",
```

Comment:
We can reuse [gulp-header](https://github.com/gerich-home/it-depends/pull/53/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R23)